### ISSUE TITLE
Allow absolute paths in mappings

### ISSUE THREAD

**Issue by iFarmGolems** [mkloubert/vs-deploy#62]
Hey!

Could you implement a feature that would allow to use absolute paths in mappings?

Currently, I can't map it "backwards" like this:

```
{
  "source": "cfg",
  "target": "../../../app/cfg"
}
```

I was also trying to use an absolute path (/opt/app/cfg) but it does not seem to work.

It would be great if I could start target path with slash and it would go from the / directory in linux.

**Comment by owenfarrell** [comment]
@iFarmGolems - Just to make sure I understand the ask, are you looking to flatten the directory structure of your `cfg` folder?

To put it another way, you don't want to respect the folder hierarchy relative to your VSCode workpace root.

**Comment by iFarmGolems** [comment]
@owenfarrell

> you don't want to respect the folder hierarchy relative to your VSCode workpace root.

Yes, basically this. But I am not to blame, it is an old project so configuration files were in different location than the "project" itself.

I don't really need this feature that much, it was just a proposal since the plugin is capable of such things.

**Comment by owenfarrell** [comment]
I think the ask is reasonable. ANT has glob-style mapping patterns to support the kind of feature you're asking for.

To extend the current target capability (effectively an identity mapper) in to other implementations (flatten, regex, etc.) would be a great feature.

Just not sure the best way to implement it.

On May 15, 2017, at 14:17, Patrik Jajcay <notifications@github.com> wrote:

@owenfarrell

you don't want to respect the folder hierarchy relative to your VSCode workpace root.

Yes, basically this. But I am not to blame, it is an old project so configuration files were in different location than the "project" itself.

I don't really need this feature that much, it was just a proposal since the plugin is capable of such things.
